### PR TITLE
packages/dev-console/src/component/project-access: tests for ProjectAccessForm

### PR DESCRIPTION
**Pull request by debsmita1**
This PR adds tests for the ProjectAccessForm component
Jira task: https://issues.redhat.com/browse/ODC-2598

### CODE REVIEW

File: frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccessForm.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        .find(Form)
+        .shallow()
+        .find(MultiColumnField)
+        .children()
+        .at(1)
+        .props().items,
+    ).toEqual({ '': 'Select a role', admin: 'Admin', view: 'View', edit: 'Edit' });
+  });
+
+  it('should have the save and reload buttons ', () => {
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccessForm.spec.tsx]: You're testing the `FormFooter` component, not yours. Drop this test as it doesn't test anything your component has the power over.

You could replace it with a simple "Renders `FormFooter`" test, but I wouldn't go any further than that.

**Comment by spadgett** [comment]
/hold
due to a bug in CI

**Pull request merged**